python/src/FunctionalChaosAlgorithm_doc.i.in

```diff
@@ -202,7 +282,58 @@ Test it:
 >>> print(model(X))
 [0.239713]
 >>> print(metamodel(X))
-[0.239514]"
+[0.239514]
+
+There are several methods to define the algorithm: default 
+settings are used when the information is not provided by the user.
+The simplest is to set only the input and output samples.
+In this case, the distribution and its parameters are estimated from
+the `inputSample` using the :class:`~openturns.MetaModelAlgorithm.BuildDistribution` class.
+See the :doc:`/auto_meta_modeling/polynomial_chaos_metamodel/plot_chaos_build_distribution`
+example for more details on this topic.
+
+>>> algo = ot.FunctionalChaosAlgorithm(inputSample, outputSample)
+
+In many cases, the distribution is known and the best is to use this information
+when we have it.
+
+>>> algo = ot.FunctionalChaosAlgorithm(inputSample, outputSample, distribution)
+
+A more involved method is to define the method to set the orthogonal basis of functions or
+polynomials.
+We use the :class:`~openturns.OrthogonalProductPolynomialFactory` class to define the
+orthogonal basis of polynomials.
+Then we use the :class:`FixedStrategy` to define the maximum number of candidate polynomials
+to consider in the expansion, up to the total degree equal to 10.
+
+>>> enumerateFunction = ot.LinearEnumerateFunction(inputDimension)
+>>> maximumTotalDegree = 10
+>>> totalSize = enumerateFunction.getBasisSizeFromTotalDegree(maximumTotalDegree)
+>>> polynomialsList = []
+>>> for i in range(inputDimension):
+...     marginalDistribution = distribution.getMarginal(i)
+...     marginalPolynomial = ot.StandardDistributionPolynomialFactory( \
+...         marginalDistribution
+...     )
```

Comment:
This should fit on a single line.

Reply:
If we do it, the line has 92 characters, 3 characters more than the expected 89 maximum number of characters. This is why I leave it as is.

Reply:
Couldn't we reduce the number of characters of the "marginalPolynomial" variable then? Use somthing like "marginalPolynom" This should remove 3 characters!

Reply:
```suggestion
...     marginalPolynomial = ot.StandardDistributionPolynomialFactory(marginalDistribution)
```